.travis.yml

```diff
@@ -0,0 +1,34 @@
+# Config file for automatic testing at travis-ci.org
+
+sudo: false
+language: python
+
+matrix:
+    include:
+      - python: 2.7
+        env: TOX_ENV=py27
+      - python: 3.4
+        env: TOX_ENV=py34
+      - python: 3.5
+        env: TOX_ENV=py35
+      - python: 3.6
+        env: TOX_ENV=py36
+      - python: 3.7
+        env: TOX_ENV=py37
+      - python: pypy
+        env: TOX_ENV=pypy
```

Comment:
I generally have no idea if tcms-api will work well on pypy. If it does, keep it but this is not something I will spend time on debugging or fixing in the future.